.github/workflows/build-and-test.yml

```diff
@@ -76,7 +76,6 @@ jobs:
           mypy --version
           mypy neuralcompression
           flake8 --version
-          flake8 neuralcompression tests projects
```

Comment:
Sorry, could you add back this line? Just delete `projects`.

Reply:
apologies for the misunderstanding